core/runtime/block_builder.hpp

```diff
@@ -54,6 +54,11 @@ namespace kagome::runtime {
     virtual outcome::result<common::Hash256> random_seed() = 0;
   };
 
+  template <class Stream>
+  Stream &operator>>(Stream &s, CheckInherentsResult& v) {
```

Comment:
CheckInherentsResult should be in primitives folder

Reply:
done